components/dot_menu/dot_menu.tsx

```diff
@@ -517,6 +622,16 @@ export class DotMenuClass extends React.PureComponent<Props, State> {
                         rightDecorator={<ShortcutKey shortcutKey='S'/>}
                         onClick={this.handleFlagMenuItemActivated}
                     />
+                    <Menu.ItemSubMenu
+                        direction={'left'}
+                        icon={Utils.getMenuItemIcon('icon-clock-outline')}
+                        id={`remind_post_${this.props.post.id}`}
+                        openUp={this.state.openUp}
+                        showMenu={!isSystemMessage}
+                        subMenu={postReminderSubMenuItems}
+                        subMenuClass={this.props.location === Locations.CENTER ? 'postReminderSubMenu' : 'postReminderSubMenuRHS'}
```

Comment:
When the window size is narrow, the submenu get hidden behind the main channel view when opened in the RHS. 

This is because of https://github.com/mattermost/dynamic-virtualized-list/blob/master/src/DynamicSizeList.js#L371

If we remove that line, then the RHS won't scroll.

The current fix is for the menu to open on top of the dot menu when opened in the RHS.

Reply:
**In center channel:**

<img width="1787" alt="Screenshot 2022-07-05 at 3 50 27 PM" src="https://user-images.githubusercontent.com/1779129/177308146-0115194a-cd6b-4d1b-b7b8-68a24612fb82.png">

(this also hides behind the LHS, if the center channel view is small enough)

**RHS hides behind center channel**
<img width="1034" alt="Screenshot 2022-07-05 at 3 53 37 PM" src="https://user-images.githubusercontent.com/1779129/177308422-7f6bdd79-3bac-48a1-9c8c-c9fb6a6587fb.png">


**With the fix, it overlaps the dot menu**
<img width="1788" alt="Screenshot 2022-07-05 at 3 50 48 PM" src="https://user-images.githubusercontent.com/1779129/177308611-503dd55f-554b-4a6e-885a-28109153eab2.png">

Reply:
Possibly a dumb suggestion from a backend person: can we apply some `!important` hackery to override the library's CSS?

Reply:
As far as I understand it, `!important` lets a parent overwrite a child's CSS. Here we want to the child to overwrite the parent, but that doesn't seem to be working, and forcing it in the parent breaks scrolling. 

I tried setting `!important` but I couldn't get it to work.